curriculum/challenges/english/25-front-end-development/lecture-understanding-the-different-types-of-testing/67db31c7ed46f430ce6c2da6.md

```diff
@@ -1,33 +1,203 @@
 ---
 id: 67db31c7ed46f430ce6c2da6
-title: What Are the Differences Between Mocking, Faking, and Stubbing? 
+title: What Are the Differences Between Mocking, Faking, and Stubbing?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: T0wWrRxHLSc
 dashedName: what-are-the-differences-between-mocking-faking-and-stubbing
 ---
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What are the differences between mocking, faking, and stubbing?
+
+In a prior lecture video, we briefly discussed what mocking is in testing. But in this video, we will dive a little bit deeper into mocking, faking and stubbing.
```

Comment:
Did we discuss mocking before?
I couldn't find that topic in the project sheet.

Reply:
Sure, this can be taken care of in another PR.